### PR TITLE
Add: idp/azure and teams

### PR DESCRIPTION
**Pull request by zmzlois**
## What's added in this PR?

- add microsoft/Azure in idp
- teams

### CODE REVIEW

File: src/components/SidebarSublist.astro
Comment (buraz82) [inline src/components/SidebarSublist.astro:1]: Can we use the term Azure Active Directory instead of Azure?

Comment (loisisar) [reply src/components/SidebarSublist.astro:1]: good shout

File: src/content/docs/installation/idps.mdx [hunk 1]
```diff
@@ -679,9 +832,301 @@ PUBLIC_REPOS REPO_READ REPO_WRITE
...
+            </tr>
+            </tbody>
+        </table>
+    </content>
+</details>
+
+### Confirm Google is working
+You can now open your Daytona dashboard on the browser and try to sign in with your Google account.
+
+## Azure
```
Comment (loisisar) [inline src/content/docs/installation/idps.mdx:996]: Change this to Azure Active Directory

**Review by nkkko — APPROVED** [review]

File: src/content/docs/configuration/teams.md [hunk 2]
```diff
@@ -1,6 +1,164 @@
 ---
 title: Teams Configuration
-description: Add a brief description of the Teams Configuration page here
+description: This guide will help you configure and manage teams in your Daytona deployment.
 sidebar:
   label: Teams
 ---
+
+Within a Daytona deployment, teams share a cohesive environment where team members can share [workspaces](/usage/workspaces), collaborate, and manage workspaces. On Daytona dashboard, you can invite, manage and configure team members, become a owner of a team, and the configuration is synced across all the [IDEs](/usage/ides) connected to your Daytona deployment.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:8]: @zmzlois this is not true.

File: src/content/docs/configuration/teams.md [hunk 3]
```diff
@@ -1,6 +1,164 @@
...
   label: Teams
 ---
+
+Within a Daytona deployment, teams share a cohesive environment where team members can share [workspaces](/usage/workspaces), collaborate, and manage workspaces. On Daytona dashboard, you can invite, manage and configure team members, become a owner of a team, and the configuration is synced across all the [IDEs](/usage/ides) connected to your Daytona deployment. 
+
+In this guide, we will cover how to configure and manage teams in your Daytona deployment.
+
+
+## Pre-requisites
+Before you start configuring teams, make sure you have the following:
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:14]: @zmzlois make sure you have access to a Daytona deployment

File: src/content/docs/configuration/teams.md [hunk 4]
```diff
@@ -1,6 +1,164 @@
...
+1. Daytona deployment
+
+Once you have a Daytona deployment, you can access the **daytona dashboard**: we will be refering to your Daytona dashboard as `https://domain.com`.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:17]: we should keep the url across docs: "https://daytona.example.com"

File: src/content/docs/configuration/teams.md [hunk 5]
```diff
@@ -1,6 +1,164 @@
...
+   
+[//]: # ({/*TODO: docs - second list item should be corrected rendered and indented*/})
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:19]: @zmzlois We need create teams first then manage teams then delete teams. Also feels like the horizonal seperater that you used on IDPs is missing

File: src/content/docs/configuration/teams.md [hunk 6]
```diff
@@ -1,6 +1,164 @@
...
+  - **Inactivity timeout**: how many minutes of inactivity before a workspace is automatically stopped.
+  - **Used minutes**: how many minutes you and your team members have used in total in the last 30 days.
+  - **Active Workspaces**: how many [active workspaces](/usage/workspaces) exits in the current time within this team.
+:::note
+  Every 30 days is a cycle for resetting the usage minutes. 
+:::
+
+## Teams on IDEs
+
+On different IDEs that utilises Daytona's [VS Code extension](/tools/vs-code-extension) or [JetBrain's Gateway Plugin](/tools/jetbrains-gateway), at the moment you will be able to:
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:152]: We dont need to link twice to the plugins. rather just an intro that you can do this via the ides and then the segments below for each

File: src/content/docs/configuration/teams.md [hunk 7]
```diff
@@ -1,6 +1,164 @@
...
+  - can't invite new team members
+  - can't assign roles to team members
+  - can't remove team members from the team
+  - can't edit team name
+
+
+### Inviting team members
+
+:::note
+At the moment, all the team related configuration can be only done on Daytona dashboard or Keycloak. You won't be able to configure teams from IDEs.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:66]: @zmzlois this is incorrect. Team are not correlated with Keycloak

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
@zmzlois Comments added.

File: src/content/docs/configuration/teams.md (hunk 7 above)
Comment (zmzlois) [reply src/content/docs/configuration/teams.md:66]: my bad.

File: src/content/docs/configuration/teams.md (hunk 2 above)
Comment (zmzlois) [reply src/content/docs/configuration/teams.md:8]: fixing

File: src/content/docs/configuration/teams.md [hunk 8]
```diff
@@ -1,6 +1,185 @@
 ---
 title: Teams Configuration
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:2]: Title should be just Teams

File: src/content/docs/configuration/teams.md [hunk 9]
```diff
@@ -1,6 +1,185 @@
...
-description: Add a brief description of the Teams Configuration page here
+description: This guide will help you configure and manage teams in your Daytona deployment.
 sidebar:
   label: Teams
 ---
+
+On Daytona dashboard, users can invite, manage and configure team members, become a owner of a team, and the configuration is synced across all your [IDEs](/usage/ides) that are connected to your Daytona deployment.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:8]: This first section is always the Overview. In the over view it should first explain what is the purpose of the feature  and then what can be done with it.

here is my edit for this:

A team represents a group of people collaborating together in Daytona, either as an internal unit like the Engineering department or as an external group, such as a team of Open Source Collaborators.

By creating a team, you become the team owner. This role empowers you to invite, manage, and configure team members, and most importantly, to manage the hardware resources at their disposal.

File: src/content/docs/configuration/teams.md [hunk 10]
```diff
@@ -1,6 +1,185 @@
...
+
+Via Daytona' team, you can: 
+
+- [Managing team members](#managing-teams)
+- [Configuring team settings](#configuring-team-settings)
+- [Check team's statistics](#teams-statistics)
+
+<hr/>
+
+## Pre-requisites
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:18]: @zmzlois you changed the way you spelled and the way you presented  "prerequisites" this should be consistant across docs.

File: src/content/docs/configuration/teams.md [hunk 11]
```diff
@@ -1,6 +1,185 @@
...
+Before you start configuring teams, make sure you have access to a **Daytona deployment** and the **dashboard URL**. 
+
+
+   
+[//]: # ({/*TODO: docs - second list item should be corrected rendered and indented*/})
+<hr/>
+
+## Managing teams
+
+In this section, we will walk you through how you can invite, assign roles, and remove team members from your daytona dashboard. Whenever you register an account on a Daytona deployment, you are default to be in a team. You can create a new team, or join an existing team.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:28]: @zmzlois  this section feels out of place. as firs you need to Create a team to manage it. Also I feel that nost of this fits in the Overview anyway

File: src/content/docs/configuration/teams.md [hunk 12]
```diff
@@ -1,6 +1,185 @@
...
+
+## Creating teams
+
+1. Log into your Daytona dashboard
+2. Creating a team
+- Click on your **team'name** in the top right corner.
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:34]: @zmzlois **team'name**  is spelt wrong. Also by default its "My Team" so you can add that in there

File: src/content/docs/configuration/teams.md [hunk 13]
```diff
@@ -1,6 +1,185 @@
...
+- In the dropdown menu, click on **Create team**.
+- Enter your team's name and click on **Create**.
+
+:::note
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:38]: This is incorrect. Some people will have a licence, so it should be informative as in "if you get an error ... if interested in an upgrade book a demo https://daytona.zapier.app/

File: src/content/docs/configuration/teams.md [hunk 14]
```diff
@@ -1,6 +1,185 @@
...
+2. Creating a team
+- Click on your **team'name** in the top right corner.
+- In the dropdown menu, click on **Create team**.
+- Enter your team's name and click on **Create**.
+
+:::note 
+At the moment you won't be able to add more than one team under your license. Feature to add more teams is coming soon.
+:::
+
+
```
Comment (ivan-burazin) [inline src/content/docs/configuration/teams.md:42]: @zmzlois I feel that here we should explain now that you create a new team or using the default my team you can mange it from the left sidebar. Under Team you will find subscription, members and settings.

And then break down the next sections

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
Added comments for review

File: src/content/docs/usage/account.md [hunk 15]
```diff
@@ -4,3 +4,18 @@ description: Add a brief description of the Account Usage page here
 sidebar:
   label: Account
 ---
+
```
Comment (ivan-burazin) [inline src/content/docs/usage/account.md:7]: @zmzlois  Account is missing the Overview, what is an account why is it here and what te user can do with it

File: src/content/docs/usage/account.md [hunk 16]
```diff
@@ -4,3 +4,18 @@ description: Add a brief description of the Account Usage page here
...
+## Managing secrets
```
Comment (ivan-burazin) [inline src/content/docs/usage/account.md:8]: Also missing the entire settings section and SSH

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
Account settings missing

File: src/content/docs/usage/account.md (hunk 15 above)
Comment (zmzlois) [reply src/content/docs/usage/account.md:7]: account is not finished. patched in next PR. just moved the secret bit there

File: src/content/docs/usage/account.md (hunk 16 above)
Comment (zmzlois) [reply src/content/docs/usage/account.md:8]: I wrote the secrets part in workspaces just moved them to accounts, guide for account is not finished

File: src/content/docs/configuration/teams.md (hunk 14 above)
Comment (zmzlois) [reply src/content/docs/configuration/teams.md:42]: I am removing the **Managing team** and have all of them separate as <h2>

**Comment by zmzlois** [comment]
Reformatted Jetbrains Gateway

File: src/content/docs/configuration/teams.md [hunk 17]
```diff
@@ -1,6 +1,204 @@
...
+- Admin
+- Member
+
+The following [different roles](#different-roles) section explains what tasks different roles can perform.
+
+### Different roles
+
+All the roles in a team can see who is in the same team and the pending invitation.
+
```
Comment (zmzlois) [inline src/content/docs/configuration/teams.md:93]: This part might better off use a table

**Pull request closed without merging**